test-templates/src/groups.rs

```diff
@@ -237,9 +237,14 @@ macro_rules! __test_group {
             let rng = &mut ark_std::test_rng();
             for _ in 0..ITERATIONS {
                 let a = Affine::rand(rng);
+                let a_group = a.into_group();
                 let b = <$group>::rand(rng);
-                assert_eq!(a + b, a.into_group() + b);
-                assert_eq!(b + a, a.into_group() + b);
+                assert!(a.is_on_curve());
+                assert!(b.into_affine().is_on_curve());
+                #[cfg(feature = "std")]
+                std::dbg!("Starting equality");
```

Comment:
is there a specific reason why we have a `dbg!` here?

Reply:
Oops, got left behind while debugging.

Reply:
Fixed.